treeherder/perf/fixtures/performance_framework.json

```diff
@@ -28,7 +28,7 @@
     "model": "perf.PerformanceFramework",
     "fields": {
       "name": "awfy",
-      "enabled": true
+      "enabled": false
```

Comment:
This is the gist of this whole PR.

Still, the frontend needs to know which frameworks are enabled/disabled. That can be tweaked [here](https://github.com/mozilla/treeherder/blob/368c112266f4f276251a4886a5337fcb17b3a1e9/treeherder/webapp/api/performance_serializers.py#L54).